src/App.tsx

```
@@ -266,5 +267,23 @@ function App() {
     </ThemeProvider>
   );
 }
+function App() {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'App'. Consider to delete it or rename it.

'App' is defined here:


(lint/suspicious/noRedeclare)